tests/snuba/api/endpoints/test_organization_stats_v2.py

```diff
@@ -276,6 +318,43 @@ def test_no_projects(self):
         assert response.status_code == 400, response.content
         assert result_sorted(response.data) == {"detail": "No projects available"}
 
+    @freeze_time("2021-03-14T12:27:28.303Z")
+    def test_open_membership_semantics(self):
+        self.org.flags.allow_joinleave = True
+        self.org.save()
+        response = self.do_request(
+            {
+                "project": [-1],
+                "statsPeriod": "1d",
+                "interval": "1d",
+                "field": ["sum(quantity)"],
+                "category": ["error", "transaction"],
+                "groupBy": ["project"],
+            },
+            user=self.user2,
+        )
+
+        assert response.status_code == 200
+        assert result_sorted(response.data) == {
+            "start": "2021-03-14T00:00:00Z",
+            "end": "2021-03-14T12:28:00Z",
+            "intervals": ["2021-03-14T00:00:00Z"],
+            "groups": [
+                {
+                    "by": {"project": self.project.id},
+                    "series": {"sum(quantity)": [6]},
+                    "totals": {"sum(quantity)": 6},
+                },
+                {
+                    "by": {"project": self.project2.id},
+                    "series": {"sum(quantity)": [1]},
+                    "totals": {"sum(quantity)": 1},
+                },
+            ],
+        }
+        self.org.flags.allow_joinleave = False
+        self.org.save()
```

Comment:
I don't think you should need to update the org at the end of the test. Each test should make a new org.

Reply:
👍🏼